docs/assemblies/proc_hotfixing-the-data-plane-container-content-images.adoc

```diff
@@ -0,0 +1,30 @@
+[id="proc_hotfixing-the-data-plane-container-content-images-{context}"]
+= Hotfixing the data plane container content with images
+
+[role="_abstract"]
+
+When container hotfix content is delivered as images, the container processes need to be restarted to use the new images. This will be accomplished by creating a new `OpenStackDataPlaneDeployment`.
```

Comment:
```suggestion
When container hotfix content is delivered as images, the container processes need to be restarted to use the new images. Create a new `OpenStackDataPlaneDeployment` custom resource (CR) that restarts the containers with the new images.
```
Active voice + removed future tense.

Reply:
this reads to me like the user needs to do something special or different in order create the deployment "that restarts...". Is there a way to say it so that the meaning is "Create a new OpenStackDataPlaneDeployment ... which will result in the containers being restarted"?

Reply:
`This restarts the containers with the new images`

Reply:
this was resolved.